docs/openfaas-pro/dashboard.md

```diff
@@ -34,7 +34,7 @@ dashboard:
   publicURL: https://dashboard.example.com
 ```
 
-The `publicURL`, doesn't necessarily have to be publicly exposed on the Internet, but it does need to be a fully qualified domain name (FQDN).
+The `publicURL`, doesn't necessarily have to be publicly exposed on the Internet, but it does need to be a fully qualified domain name (FQDN). It can be left blank if you want to access the dashboard using port-forwarding.
```

Comment:
What are valid options in addition to `` empty string?

Reply:
I think we need to improve this section so that it's presented as a third option in its own right along with the port-forwarding command required and a sample `values.yaml` to make it obvious what we mean by "empty".

```
  publicURL: ""
```